projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/lang/zh_cn.json

```
@@ -556,21 +556,21 @@
     "programmingPuzzle.pneumaticcraft.condition_item.name": "条件：物品过滤器",
     "gui.tooltip.item.pneumaticcraft.logistics_configurator": "此物品可以用于配置物流框架和传输装置。潜行右击以移除框架和其他可附加物（导热框架，作物架...）",
     "gui.tooltip.item.pneumaticcraft.pneumatic_leggings": "该护腿在升级后会加快你的移速。",
-    "gui.tooltip.item.pneumaticcraft.memory_stick": "存储玩家经验。${br}▶右击：存储一级${br}▶潜行右击：取出一级${br}▶左键单击：切换为自动吸收经验球",
-    "gui.tooltip.item.pneumaticcraft.stop_worm": "用于安全站中提升骇入的几率。",
-    "gui.tooltip.item.pneumaticcraft.remote": "可以通过放置按钮、标签和其他GUI组件来创建一个用来控制无人机或通用传感器的遥控器！",
+    "gui.tooltip.item.pneumaticcraft.memory_stick": "存储玩家经验。${br}▶右击：存储一级${br}▶潜行右击：取出一级${br}▶左击：切换为自动吸收经验球",
+    "gui.tooltip.item.pneumaticcraft.stop_worm": "用于安全站中提升侵入的几率。",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
    "gui.tooltip.item.pneumaticcraft.stop_worm": "用于安全站中，可以提升侵入的几率。",
```